.github/workflows/scripts/build-dependencies-macos.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
Just a note that the VulkanSDK now sets the macOS deployment target minimum to 11.0. 10.13 is getting rather old.

Reply:
Okay, I updated the script